allianceutils/models.py

```diff
@@ -29,15 +29,15 @@ def combine_querysets_as_manager(*queryset_classes: List[QuerySet]) -> Manager:
 
 class NoDeleteQuerySet(QuerySet):
     def delete(self, force: bool=False):
-        raise IntegrityError("Instances of model '%s' are marked as undeletable" % self.__class__.__name__)
+        raise IntegrityError(f"Instances of model '{self.__class__.__name__}' are marked as undeletable")
 
 
 class NoDeleteModel(Model):
     """
     A model that cannot be deleted.
 
     Note that this is an abstract Model, please read
-    https://docs.djangoproject.com/en/1.11/topics/db/managers/#custom-managers-and-model-inheritance
+    https://docs.djangoproject.com/en/dev/topics/db/managers/#custom-managers-and-model-inheritance
```

Comment:
probably should use `stable` rather than `dev`

Reply:
I didn't know that existed -- was wondering why I'd never seen it and realised it's because it's a redirect.

Have updated all links in the project.